tests/org.dataflowanalysis.analysis.converter.tests/src/org/dataflowanalysis/analysis/converter/tests/ConverterTests.java

```diff
@@ -0,0 +1,303 @@
+import org.dataflowanalysis.analysis.pcm.PCMDataFlowConfidentialityAnalysisBuilder;
+import org.dataflowanalysis.analysis.pcm.core.seff.*;
+import org.dataflowanalysis.analysis.pcm.core.user.*;
+import org.dataflowanalysis.analysis.testmodels.Activator;
+import org.dataflowanalysis.dfd.datadictionary.Assignment;
+import org.dataflowanalysis.dfd.datadictionary.DataDictionary;
+import org.dataflowanalysis.dfd.datadictionary.Pin;
+import org.dataflowanalysis.dfd.dataflowdiagram.DataFlowDiagram;
+import org.dataflowanalysis.dfd.dataflowdiagram.Flow;
+import org.dataflowanalysis.dfd.dataflowdiagram.Node;
+import org.dataflowanalysis.dfd.dataflowdiagram.dataflowdiagramPackage;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+
+public class ConverterTests {
+	private static final String packagePath="src/org/dataflowanalysis/analysis/converter/tests/";
+	
+	@Test
+	@DisplayName("Test Web -> DFD -> Web")
+	public void webToDfdToWeb() {
+		Main.readWeb(packagePath+"minimal.json");
+		Main.readDFD(packagePath+"minimal",packagePath+"test.json");
+		
+		ObjectMapper objectMapper = new ObjectMapper();        
+		File file = new File(packagePath+"minimal.json");
+		DFD webBefore = null;
+        try {
+            webBefore = objectMapper.readValue(file, DFD.class);
+        } 
+        catch (IOException e) {}
+        
+        objectMapper = new ObjectMapper();        
+		file = new File(packagePath+"test.json");
+		DFD webAfter = null;
+        try {
+            webAfter = objectMapper.readValue(file, DFD.class);
+        } 
+        catch (IOException e) {}
+                
+        webBefore.labelTypes().sort(Comparator.comparing(WebLabelType::id));
+        webAfter.labelTypes().sort(Comparator.comparing(WebLabelType::id));
+        
+        List<Child> childrenBefore = webBefore.model().children();
+        List<Child> childrenAfter = webAfter.model().children();
+
+        childrenBefore.sort(Comparator.comparing(Child::id));
+        childrenAfter.sort(Comparator.comparing(Child::id));
+                
+        List<Child> combined=new ArrayList<>(childrenBefore);
+        combined.addAll(childrenAfter);
+        for(Child child: combined) {
+        	if(child.labels()!=null) {
+            	child.labels().sort(Comparator.comparing(WebLabel::labelTypeId).thenComparing(WebLabel::labelTypeValueId));
+        	}
+        	if(child.ports() != null) {
+            	child.ports().sort(Comparator.comparing(Port::id));
+
+        	}
+        }
+        
+        assertEquals(webBefore,webAfter);
+        
+        cleanup(packagePath+"minimal.datadictionary");
+        cleanup(packagePath+"minimal.dataflowdiagram");
+        cleanup(packagePath+"test.json");
+	}
+	
+	@Test
+	@DisplayName("Test Micro -> DFD")
+	public void microToDfd() {
```

Comment:
I would opt for moving the tests of the different converters to individual classes to enhance to readability as this is quite some code for one test class